hmns-app/client/app/(tabs)/index.tsx

```diff
@@ -1,19 +1,167 @@
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod adipiscing elit, sed do eiusmod
+						</Text>
+						<Pressable className="p-2 px-3 rounded-full self-start" style={{backgroundColor: colors.learnmore}}>
+							<Text className="text-xs">Learn more</Text>
+						</Pressable>
+					</View>
+					{/* Right Side */}
+					<View className="flex bg-transparent w-2/5">
+						<View className="bg-white w-full h-4/5 rounded-lg flex justify-around items-center">
+							<Text className="text-black">Photo</Text>
+						</View>
+					</View>
+				</View>
+
+				{/* Challenges */}
+				<View className="flex bg-transparent" style={{gap: 20}}>
+					{/* Challenges Header */}
+					<View className="flex flex-row justify-between bg-transparent">
+						<Text className="text-xl font-bold" style={{color: colors.fossilRim}}>Challenges</Text>
+						<Pressable className="flex justify-around items-center">
+							<Text className="text-black">View all</Text>
+						</Pressable>
+					</View>
+
+					{challenges.map(challenge => (
+						<ChallengeItem key={challenge.title} challenge={challenge} />
+					))}
+				</View>
+
+				{/* Resources */}
+				<View className="flex bg-transparent" style={{gap: 20}}>
+					{/* Resources Header */}
+					<View className="flex flex-row justify-between bg-transparent">
+						<Text className="text-xl font-bold" style={{color: colors.fossilRim}}>Resources</Text>
+						<Pressable className="flex justify-around items-center">
+							<Text className="text-black">View all</Text>
+						</Pressable>
+					</View>
+
+					{resources.map(resource => (
+						<ResourceItem key={resource.title} resource={resource} />
+					))}
+				</View>
+			</ScrollView>
+		</SafeAreaView>
+	);
+}
+
+const ChallengeItem = ({challenge}: {challenge: Challenge}) => {
+	return (
+		<View className="flex flex-row rounded-2xl p-4 border items-center justify-between bg-transparent">
+			<View
+				style={{
+					backgroundColor: "transparent",
+					borderRadius: 9999,
+					width: 30,
+					height: 30,
+					borderColor: "black",
+					borderWidth: 2,
+				}}
+			/>
+			<View className="bg-transparent ml-2">
+				<Text className="text-black flex-wrap font-bold text-base">{challenge.title}</Text>
+				<Text className="text-black flex-wrap">{challenge.content}</Text>
+			</View>
+		</View>
+	);
+};
+const ResourceItem = ({resource}: {resource: Resource}) => {
+	return (
+		<View className="flex flex-row rounded-2xl p-4 border items-start justify-between bg-transparent">
+			<View className="bg-black w-1/3 h-28 rounded-lg flex justify-around items-center">
+				<Text className="text-white">Photo</Text>
+			</View>
+			<View className="bg-transparent ml-2" style={{maxWidth: "64%"}}>
+				<Text className="text-black flex-wrap font-bold text-base">{resource.title}</Text>
+				<Text className="text-black flex-wrap">{resource.content}</Text>
+			</View>
+		</View>
+	);
+};
 
-  return (
+  /*return (
```

Comment:
In the process of resolving merge conflicts, I commented out this piece of code. This code is responsible for displaying the button required to open the modal. The integration of this button with the larger homepage can be done in a new PR as a separate task. I won't mind doing this.